docs/topics/native/native-get-started.md

```diff
@@ -1,42 +1,68 @@
-[//]: # (title: Get started with Kotlin/Native in IntelliJ IDEA)
+[//]: # (title: Get started with Kotlin/Native)
 
-This tutorial demonstrates how to use IntelliJ IDEA for creating a Kotlin/Native application.
+In this tutorial, you'll learn how to create a Kotlin/Native application.
 
-To get started, install the latest version of [IntelliJ IDEA](https://www.jetbrains.com/idea/download/index.html).
-The tutorial is applicable to both IntelliJ IDEA Community Edition and the Ultimate Edition.
+With Kotlin/Native, you can compile for [different targets](native-target-support.md), including Linux, macOS, and Windows.
+While cross-platform compilation is possible, which means using one platform to compile for a different one,
+in this case, you'll be targeting the same platform you're compiling on.
 
-> If you have a Mac with macOS and want to create and run iOS or desktop applications, you also need to
-> [install Xcode Command Line Tools](https://developer.apple.com/download/). For more information,
-> see [Set up an environment](https://www.jetbrains.com/help/kotlin-multiplatform-dev/multiplatform-setup.html).
-> 
+There are several options for getting started with Kotlin/Native:
+
+* The easiest way to visualize the project creation is to [use an IDE](#in-ide).
+* If you want to get a better understanding of how things work under the hood, [create build files for the project manually](#using-gradle).
+  
+  The default choice for Kotlin/Native and Kotlin Multiplatform is the [Gradle](https://gradle.org) build system,
+  commonly used in Java, Android, and other ecosystems.
+
+* You can also use the Kotlin/Native compiler (which is shipped as a part of the standard Kotlin distribution)
+  [directly in the command line tool](#using-the-command-line-compiler).
+
+  While console compilation seems easy and clear, it doesn't scale well for larger projects with hundreds of files and
+  libraries. For such projects, it's recommended to use an IDE or a build system.
+
+> If you use a Mac and want to create and run applications for macOS or other Apple targets, you also need to
+> install [Xcode Command Line Tools](https://developer.apple.com/download/), launch it, and accept the license terms first.
+>
 {style="note"}
 
-## Before you start
+## In IDE
+
+In this section, you'll learn how to use IntelliJ IDEA to create a Kotlin/Native application. You can use both
+the Community Edition and the Ultimate Edition.
+
+### Create the project
 
 1. Download and install the latest version of [IntelliJ IDEA](https://www.jetbrains.com/idea/) with the latest [Kotlin plugin](releases.md).
```

Comment:
Since the plugin is now bundled in IntelliJ IDEA, should we not mention it anymore? 🤔

Reply:
True, forgot about that. thanks!